src/App.js

```diff
@@ -1,13 +1,14 @@
 import "./App.css";
 
-import HomePage from "./pages/HomePage/HomePage";
-import Footer from "./components/Footer/Footer";
-import NavBar from "./components/NavBar/NavBar";
+import { Footer } from "./components/Footer/Footer";
+import { NavBar } from "./components/NavBar/NavBar";
+import { RoutersPath } from "./Routers/RoutersPath";
+
```

Comment:
you can create an index file and export all components there.
![image](https://user-images.githubusercontent.com/30016242/160195071-75ace5a9-93be-440e-b270-1745cc993de8.png)
![image](https://user-images.githubusercontent.com/30016242/160195108-d46bcae5-57e8-4473-bde7-87da7a3d2c74.png)



while importing you can use single line import. 
![image](https://user-images.githubusercontent.com/30016242/160195183-18cc8d4c-8c67-496a-9af1-465d7464da7c.png)

Reply:
You can also read about path aliasing. I haven't done that yet.

Reply:
I will do that, thank